### ISSUE TITLE
node.type \n should be newlines or whitespace

### ISSUE THREAD

**Issue by milahu** [tree-sitter/tree-sitter-bash#107]
i get these nodes from empty lines

```js
{ type: '\n', text: '\n\n' }
{ type: '\n', text: '\n' }
```

the node.type should be printable, for example

```js
{ type: 'newlines', text: '\n\n' }
// or
{ type: 'whitespace', text: '\n' }
```

**Comment by amaanq** [comment]
newline/whitespace have different meanings on different platforms/standards, (\r, \n, \r\n, etc for newlines)

maybe LF or CR is a better alternative

**Comment by amaanq** [comment]
hm, not sure how you got this, is this via the node bindings? either way, I dont think the problem directly lies here, but would be good to know more

**Comment by ahlinc** [comment]
@amaanq  I think this also was fixed in #198, because no more visible anonymous `\n` are exposed.
